tests/rkt_tests.go

```diff
@@ -312,6 +312,31 @@ func runRktAsUidGidAndCheckOutput(t *testing.T, rktCmd, expectedLine string, exp
 	}
 }
 
+func runRkt(t *testing.T, rktCmd string, uid, gid int) (string, int) {
+	child, err := gexpect.Command(rktCmd)
+	if err != nil {
+		t.Fatalf("cannot exec rkt: %v", err)
+	}
+	if gid != 0 {
```

Comment:
I stumbled upon this myself ;-) the reasoning is explained here https://github.com/coreos/rkt/commit/2df2c7b9111c4d7d08ae75550bc6f7cd42f14532 and was introduced in this very commit https://github.com/coreos/rkt/commit/2df2c7b9111c4d7d08ae75550bc6f7cd42f14532#diff-d70993595ae1ece0fcf84c06a2694737R327.